drivers/volume/volume.go

```diff
@@ -70,6 +70,13 @@ type Driver interface {
 
 	// GetStorageDevices returns the list of storage devices used by the given node.
 	GetStorageDevices(n node.Node) ([]string, error)
+
+	//GetVolumeReplicationFactor returns the current replication factor of the volume.
+	//TODO - Can we define the return type to be of the type *api.VolumeSpec.HaLevel?
```

Comment:
We don't need to do this TODO as api.VolumeSpec.HaLevel is very OSD specific. Let's keep this as generic as possible. int64 seems fine for now, unless we decide to only support OSD.
nit: spaces after // .. for SetRF as well